RELEASE.md

```diff
@@ -1,3 +1,24 @@
+# Release 2.1.0
+
+## New Features
+- Search result maximum limit and offset greatly increased. Maximum `limit` parameter increased from 400 to 1,000, `offset` increased from 1,000 to 10,000. Maximum value for `MARQO_MAX_RETRIEVABLE_DOCS` configuration is now 10,000 (https://github.com/marqo-ai/marqo/pull/735, https://github.com/marqo-ai/marqo/pull/737​​). See search limit and offset usage [here](https://docs.marqo.ai/2.1.0/API-Reference/Search/search/#limit)
+
+## Bug fixes and minor changes
+- Improved the Marqo bootstrapping process to address unexpected API behaviour when no index has been created yet (https://github.com/marqo-ai/marqo/pull/730).
+- Addressed a bug where create_index accepted both dependent_fields and dependentFields (https://github.com/marqo-ai/marqo/pull/717, https://github.com/marqo-ai/marqo/pull/734). Using dependent_fields will now raise a 400 validation error.
+- Fixed parsing issue for documents in unstructured indexes (https://github.com/marqo-ai/marqo/pull/732). 
```

Comment:
Again, can we focus on the value this change is bringing to users, rather than the negatives

Reply:
e.g.: "improved validation for ..."